### PR TITLE
Create juliacon-highlights

### PR DESCRIPTION
**Pull request by pakzy**

### CODE REVIEW

**Comment by johnnychen94** [comment]
Maybe with `.md` extension?

**Comment by ViralBShah** [comment]
@logankilpatrick How does this look? Any additions?

File: blog/2021/08/juliacon-highlights.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+@def title =  "JuliaCon 2021 Highlights"
+@def authors = "The JuliaCon 2021 committee"
+@def published = "20 August 2021"
+@def rss_pubdate = Date(2021, 8, 20)
+@def rss = """JuliaCon 2021 Highlights"""
+
+This year’s [JuliaCon](https://www.youtube.com/playlist?list=PLP8iPy9hna6Q343_8sSq4f306VGLW4TLK) was the biggest and best ever, with [more than 300 presentations available for free on YouTube](https://www.youtube.com/playlist?list=PLP8iPy9hna6Q343_8sSq4f306VGLW4TLK), more than 20,000 registrations, and more than 43,000 unique YouTube viewers during the conference.
```
Comment (giordano) [inline blog/2021/08/juliacon-highlights.md:7]: Can we have a comparison with numbers from last year, to show an increase (there has been an increase, right? :sweat_smile:)

Comment (logankilpatrick) [reply blog/2021/08/juliacon-highlights.md:7]: https://julialang.org/blog/2020/08/juliacon-2020-wrapup/

**Comment by ViralBShah** [comment]
Please merge when ready.

**Comment by ViralBShah** [comment]
These folks have all done some JuliaCon video aggregations: sciml, Josh Day's data science blog, julia computing, and invenia. Should we link?

**Comment by logankilpatrick** [comment]
yes, though the invenia one is focused on their talks IRC

**Comment by ViralBShah** [comment]
As is julia computing's, and sciml, and Josh's data science blog. they are just different aggregations.

**Comment by pfitzseb** [comment]
This might've just broken the website.

**Comment by ViralBShah** [comment]
Yeah the blog is broken now.

**Pull request merged**